components/libminchassis/mc_psi_system_service.cc

```diff
@@ -24,6 +24,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA */
 #include <mysql/components/service.h>
 #include <mysql/components/service_implementation.h>
 #include <mysql/components/services/mysql_psi_system_service.h>
+#include <mysql/components/services/psi_memory_service.h>
```

Comment:
I don't really like the idea of changing `minchassis` - this is a very base-level Oracle MySQL Server component

Reply:
Without these changes, the keyring migration tools will fail. I didn't notice it at first because we can't run those tests with keyring_kmip (requires backing up/restoring the keyring data and/or running multiple independent keyrings at the same time), but jenkins runs showed keyring_file test failures.

The issue is that now the keyring components depend on psi_memory, and the keyring tools can't load them without providing a dummy psi_memory implementation.

The alternative would be providing psi_memory as part of the keyring client tools, which would require duplicating significant parts of minchassis there.